README.md

```diff
@@ -2,11 +2,37 @@
 
 A Survey of:
 
-_Does the first programming language someone learns influence whether they prefer R or Python (or have no preference)?_
+_Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._
 
 | Team Members | Github Handle |
 |--------------|---------------|
 | Brenden Everitt | everittB |
 | Miliban Keyim | mkeyim |
 | Maninder Kohli | ksm45 |
-| Phuntsok Tseten | phuntsoktseten |  
+| Phuntsok Tseten | phuntsoktseten |
+
+
+#### Chosen Questions:
+* Which language do you prefer for data analysis:
+- Python or R. This is the response question.
+* What is the first computing language you learned?
+  -  This survey question explores the influence of specific languages for the preference of Python or R.
+
+* What was your undergraduate degree in? 
+  - Educational background will influence what programming language one prefers to learn first.
+
+* Are your analysis projects typically small(10K rows), medium(10-1M Rows), big (1m+ rows) ?
+  - The size of a data analysis project might influence the language a person to use because of difference in speed and ease of setup/use.
+  
+* Which language do you think has better documentation when it comes to Data Analysis?
+  - We believe having clear and concise documentation aids in the ease of applying data analysis principles and decreases the learning curve of the language. If clear examples of analysis using the languages of R or Python are not given, the user might tend to prefer to choose the one they believe has better documentation that helps them through the process.  
+
+#### Analysis of the survey results we plan to employ.
+
+We will use Logisitic regression as we have a binary outcome of either Python or R, with that we are able to see the effect of the confounding variables. 
```

Comment:
```suggestion
We plan to use logistic regression to analysis how someone's preference to use R or Python for a data analysis task is influenced by the first programming language learned they have learned, while also controlling for confounding effects. We chose a logisitic regression because we have a binary response variable, either R or Python, and will the other survery question answers as explanatory variables.
```